internal/archive/archive_test.go

```diff
@@ -335,22 +335,67 @@ func (s *httpSuite) TestArchiveLabels(c *C) {
 	}
 
 	_, err = archive.Open(&options)
-	c.Assert(err, IsNil)
+	c.Assert(err, ErrorMatches, `.*\bno Ubuntu section`)
+}
 
-	s.prepareArchiveAdjustRelease("jammy", "22.04", "amd64", []string{"main", "universe"}, setLabel("ThirdParty"))
+var proArchiveInfo = map[string]struct {
```

Comment:
In my opinion it is normally a bad sign when you have to copy some functionality from the code to the tests. Maybe we can use `export_test` to make the map in `archive.go` public or we can make it public directly. I don't think the latter is useful but I am not sure.

Reply:
Good point. Used `export_test`.